server/src-lib/Hasura/GraphQL/Resolve/Select.hs

```diff
@@ -56,7 +65,10 @@ fromSelSet f fldTy flds =
       _ -> do
         fldInfo <- getFldInfo fldTy fldName
         case fldInfo of
-          Left colInfo -> return $ RS.FCol colInfo
+          Left colInfo -> if (null $ _fArguments fld)
```

Comment:
We may not require `(null $ _fArguments fld)` check if we use `argsToColOp` kind of function mentioned in previous comment.